media/src/objects/Function.svelte

```diff
@@ -1080,21 +1137,30 @@
                 on:input={updateSurface}
                 class="box box-2"
             />
-            <span class="box-1">
-                Levels
-            </span>
-            <span class="box-2">
-                <button class="btn box-2" on:click={activateLevelElevator}>
+            <span class="box-1"> Levels </span>
+            <label class="switch box box-3">
+                <input
+                    type="checkbox"
+                    name="showLevels"
+                    id="showLevels"
+                    bind:checked={showLevelCurves}
+                />
+                <span class="slider round" />
+            </label>
+            <span class="box-4">
+                <button
+                    class="btn box-4"
```

Comment:
This button needs a `btn-primary` or `btn-secondary` bootstrap class added in order to be visible. As is, I can't really see it: 
![Screenshot_2023-02-11_10-29-34](https://user-images.githubusercontent.com/59292/218266803-8f939c73-5e84-42d7-bf73-43b5dca03eed.png)